CHANGES.md

```diff
@@ -16,6 +16,7 @@ Release Notes.
 * Refactor kotlin coroutine plugin with CoroutineContext.
 * Fix OracleURLParser ignoring actual port when :SID is absent.
 * Change gRPC instrumentation point to fix plugin not working for server side.
+* Fix servicecomb plugin trace break.
```

Comment:
Please add the GitHub issue link in this CHANGES

Reply:
We don't recommend this, as the git commit shows the relationship.